### ISSUE TITLE
How to edit with text and interpolate states as described in the paper?

### ISSUE THREAD

**Issue by G-force78** [yuangan/EAT_code#23]

**Comment by yuangan** [comment]
We interpolate two emotions between corresponding emotion prompts. 
![image](https://github.com/yuangan/EAT_code/assets/12268263/ab828ec6-b1e4-4bed-a473-6e35ddcd545e)
You need to fine-tune EAT with an additional CLIP loss to edit videos with text.
![image](https://github.com/yuangan/EAT_code/assets/12268263/480e9ae5-42ea-4317-9790-a3d41df60b85)

**Comment by JamesLong199** [comment]
Hi, 

Could you kindly provide more details on the **_setting for model fine-tuning with CLIP_** and the **_zero-shot text-guided expression editing procedure_**?

For model fine-tuning with CLIP, my understanding is that:
the same losses in emotion adaptation are used in addition to CLIP loss; 
the fine-tuning is performed on MEAD, where each training video is paired with fixed text prompts of the corresponding emotion category (attached in the screenshot).

For the zero-shot text-guided expression editing, I was wondering how is the CLIP text feature incorporated into the existing model structure.

Thank you in advance for your time and help!

![image](https://github.com/yuangan/EAT_code/assets/66498825/dc7a5698-237f-426a-bbab-6534c3bdde83)
